bigtable/google/cloud/bigtable/cluster.py

```diff
@@ -47,35 +42,97 @@ class Cluster(object):
     :param instance: The instance where the cluster resides.
 
     :type location_id: str
-    :param location_id: ID of the location in which the cluster will be
-                        created. For e.g "us-central1-b"
-                        List of locations id is :
+    :param location_id: (Creation Only) The location where this cluster's
+                        nodes and storage reside . For best performance,
+                        clients should be located as close as possible to
+                        this cluster.
+                        For list of supported locations refer to
                         https://cloud.google.com/bigtable/docs/locations
-                        For best performance, clients should be located
-                        as close as possible to this cluster.
+
+    :type state: int
+    :param state: (`OutputOnly`)
+                  The current state of the cluster.
+                  Possible values are represented by the following constants:
+                  :data:`google.cloud.bigtable.enums.Cluster.State.NOT_KNOWN`.
+                  :data:`google.cloud.bigtable.enums.Cluster.State.READY`.
+                  :data:`google.cloud.bigtable.enums.Cluster.State.CREATING`.
+                  :data:`google.cloud.bigtable.enums.Cluster.State.RESIZING`.
+                  :data:`google.cloud.bigtable.enums.Cluster.State.DISABLED`.
```

Comment:
Given that we only expect `state` to be passed in from the `from_pb` factory below, can you please rename it `_state` and put it at the end of the list of parameters?

Reply:
Done